api/Element.json

```diff
@@ -4115,6 +4115,475 @@
           }
         }
       },
+      "pointercancel_event": {
+        "__compat": {
+          "description": "<code>pointercancel</code> event",
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Element/pointercancel_event",
+          "spec_url": [
+            "https://w3c.github.io/pointerevents/#dom-globaleventhandlers-onpointercancel",
```

Comment:
Can you reverse the order of these? The first one is the more interesting reference. (I've put the handler definitions last in other PRs I've written/edited.)

Reply:
Yes, exactly!